CONTRIBUTING.rst

```diff
@@ -156,6 +156,46 @@ Opening a PR will trigger a suite of tests and style/formatting checks that must
 We also require approval from at least one (ideally multiple) of the main DESC developers, who may have suggested changes
 or edits to your PR.
 
+What if the ``test_compute_everything`` test fails, or there is a conflict in ``master_compute_data.pkl``?
+----------------------------------------------------------------------------------------------------------
+When the outputs of the compute quantities tested by the`test_compute_everything` [test](https://github.com/PlasmaControl/DESC/blob/master/tests/test_compute_funs.py) are changed in a PR, that test will fail.
+The three main reasons this could occur are:
+
+-  The PR was not intended to change how things are computed, but messed up something unexpected and now the compute quantities are incorrect, if you did not expect these changes in the PR then look into why these differences are happening and fix the PR.
+-  The PR updated the way one of the existing compute index quantities are computed (either by a redefinition or perhaps fixing an error present in ``master``)
+-  The PR added a new class parametrization (such as a new subclass of ``Curve`` like ``LinearCurve`` etc)
```

Comment:
Does this actually make the test fail? I don't think it will, but you still have to remember to add that new parameterization to the test as you explained. Also this applies to adding entirely new classes, not just new parameterizations of an existing class

Reply:
Also "parameterization" is missing a second "e"

Reply:
I think my spelling of parametrization is right or at least is an accepted spelling: https://en.wikipedia.org/wiki/Parametrization_(geometry)#:~:text=Parametrization%20is%20a%20mathematical%20process,some%20independent%20quantities%20called%20parameters.

Reply:
and yea it does fail bc of this line in `test_compute_everything`

```python
assert things.keys() == data_index.keys(), (
        f"Missing the parameterizations {data_index.keys() - things.keys()}"
        f" to test against master."
    )
```